ui/app/AppLayouts/Chat/popups/RequestPaymentModal.qml

```diff
@@ -0,0 +1,228 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+import QtQml.Models 2.15
+
+import StatusQ 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components.private 0.1 as SQP
+import StatusQ.Components 0.1
+import StatusQ.Popups.Dialog 0.1
+
+import AppLayouts.Wallet.controls 1.0
+import AppLayouts.Wallet.adaptors 1.0
+
+import shared.popups.send.views 1.0
+import shared.controls 1.0
+import shared.stores 1.0 as SharedStores
+import utils 1.0
+
+StatusDialog {
+    id: root
+
+    required property SharedStores.RequestPaymentStore store
```

Comment:
WDYT of removing the store dependency here and replace it with properties? Would be much clearer what is needed for this Modal.

I see we're not using many properties/functions from the store and could be easely done.